client/state/selectors/is-automated-transfer-enabled-for-site.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+import { abtest } from 'lib/abtest';
+import { PLAN_BUSINESS } from 'lib/plans/constants';
+import { getSitePlanSlug } from 'state/sites/selectors';
+import { canCurrentUser } from 'state/selectors';
+
+/**
+ * Returns true if Automated Transfer is enabled for the site and current user.
+ * @param   {Object}  state   The global state tree
+ * @param   {Number}  siteId  The guid of the site
+ * @returns {Boolean} true if enabled for the site and current user
+ */
+export default function isAutomatedTransferEnabledForSite( state, siteId ) {
+	// Feature must be enabled on environment
+	if ( ! isEnabled( 'automated-transfer' ) ) {
+		return false;
+	}
+
+	// Site has Business plan
+	const planSlug = getSitePlanSlug( state, siteId );
+	if ( planSlug !== PLAN_BUSINESS ) {
+		return false;
+	}
+
+	// Current User can manage site
+	const canManageSite = canCurrentUser( state, siteId, 'manage_options' );
+	if ( ! canManageSite ) {
+		return false;
+	}
+
+	// Gate to 40% roll. If we modify this value, we need a new test name
+	// in active-tests.js
+	return abtest( 'automatedTransfer1' ) === 'enabled';
```

Comment:
We're only enable this for a percentage of users, so after checking all of the other checks, we also look to see whether the user is in the pool of people who are getting this variant.